phing/tasks/properties.xml

```diff
@@ -56,7 +56,43 @@
     </then>
   </if>
 
-  <echo level="verbose">Executing commands against multisite "${multisite.name}"</echo>
+  <!-- Set the default site to operate against. -->
+  <property name="site.name" value="default" override="false"/>
+
+  <!-- Load multisite-specific overrides. -->
+  <if>
+    <available file="${blt.config-dirs.sites}"/>
+    <then>
+      <!-- Build the site list from *.yml files in the blt/sites directory. -->
+      <exec dir="${blt.config-dirs.sites}" command="ls *.yml | tr -d '\n' | sed s/\.yml/,/g" outputProperty="multisite.names"/>
```

Comment:
I think we may need a more blt-specific pattern, since this could pick up unrelated yml files. Perhaps project.site.yml.